web/frontend/src/components/slam-2d-render.vue

```diff
@@ -63,25 +72,96 @@ controls.enableRotate = false;
 
 const raycaster = new MouseRaycaster({ camera, renderer, recursive: false });
 
-raycaster.on('click', (event) => {
+raycaster.on('click', (event: THREE.Event) => {
   const [intersection] = event.intersections as THREE.Intersection[];
   if (intersection && intersection.point) {
-    console.log(intersection.point);
+    emit('click', intersection.point);
   }
 });
 
-const markerSize = 0.5;
-const marker = new THREE.Mesh(
-  new THREE.PlaneGeometry(markerSize, markerSize).rotateX(-Math.PI / 2),
-  new THREE.MeshBasicMaterial({ color: 'red' })
-);
-marker.name = 'Marker';
-// This ensures the robot marker renders on top of the pointcloud data
-marker.renderOrder = 999;
+const guiData = {
+    drawFillShapes: true,
+    drawStrokes: true,
+    fillShapesWireframe: false,
+    strokesWireframe: false,
+  };
+/*
+ * svgLoader example for webgl:
+ * https://github.com/mrdoob/three.js/blob/master/examples/webgl_loader_svg.html
+ */
+//
+const svgLoader = new SVGLoader(); 
+const makeMarker = async (url : string, name: string, scalar: number) => {
+  const data = await svgLoader.loadAsync(url);
+
+  const { paths } = data!;
+
+  const group = new THREE.Group();
+  group.scale.multiplyScalar(scalar);
+  group.position.set(-70, 0, 70) // why do we do this? - do we need it?
```

Comment:
@micheal-parks

Reply:
confirming that 
 ```
 group.position.set(-70, 0, 70)
 group.yscale.y *= -1;
 ```
 when commented out do not affect the display of the base or destination marker.
 
 i am unsure about what these two lines do and since they seem to have no effect on anything I think we should get rid of them.

Reply:
go for it